src/scheduler/Unpacking.py

```diff
@@ -115,24 +114,4 @@ def _get_combined_analysis_workload(self):
         return 0
 
     def check_exceptions(self):
-        return_value = False
-        for worker in self.workers:
-            if worker.exception:
-                logging.error("{}Worker Exception Found!!{}".format(bcolors.FAIL, bcolors.ENDC))
-                logging.error(worker.exception[1])
-                terminate_process_and_childs(worker)
-                self.workers.remove(worker)
-
-                if self.config.getboolean('ExpertSettings', 'throw_exceptions'):
-                    return_value = True
-                else:
-                    process_index = worker.name.split('-')[2]
-                    self._start_single_worker(process_index)
-        return return_value
-
-    def _start_single_worker(self, process_index):
-        process = ExceptionSafeProcess(target=self.unpack_worker,
-                                       name='Unpacking-Worker-{}'.format(process_index),
-                                       args=(process_index,))
-        process.start()
-        self.workers.append(process)
+        return check_worker_exceptions(self.workers, 'Unpacking', self.config, self.unpack_worker)
```

Comment:
Unpacker does not "die" even if config parameter that tell it to is set

Reply:
`self.workers.append()` was missing so the check ran on an empty list ... should be fixed now